packages/minter-contracts/README.md

```diff
@@ -116,6 +116,15 @@ E.g.,
 yarn michelson-to-ts -c tezos-client
 ```
 
+### Generate Contract Types (this is run as part of michelson-to-ts)
```

Comment:
To follow suit with other commands, the command name should be the header.
```markdown
### `yarn generate-types`
```

And the table of contents should be updated to include this as well.